packages/profile/src/views/Profile.jsx

```diff
@@ -56,13 +67,46 @@ const reducer = (state, action) => {
 const change = (dispatch, type, e) => {
   try {
     const target = e.target
+
+    if (type === 'avatar') {
+      onAvatarChange(dispatch, target)
+      target.value = null // reset so onChange will fire even for same image
```

Comment:
the dom cannot directly be manipulated in react. 

I recommend you read up on controlled vs uncontrolled components:
https://reactjs.org/docs/uncontrolled-components.html